lib/matplotlib/axes/_base.pyi

```diff
@@ -53,6 +56,9 @@ class _AxesBase(martist.Artist):
     transData: Transform
     ignore_existing_data_limits: bool
     axison: bool
+    containers: list[Container]
+    callbacks: CallbackRegistry
+    child_axes: list[_AxesBase]
```

Comment:
Looking at `__clear`, I see also `legend_` and `title` that aren't here; should they be added as well? (The former appears in `axes.pyi` on `Axes`)

Reply:
I'd say those ones (and perhaps several of the existing attrs) should be made private (`title` there is already `_[left|right]_title` which _are_ private, though the getters/setters are defined in `_axes.py`, not `_base.py`, for some reason, and not defined for `secondary_axes`, even though the attrs exist)

But I'll add them since they are by naming convention public (though I will also note that none of these attrs (aside from `dataLim` and `viewLim`) actually appear in the rendered docs, though several (`trans[Axes,Scale,Limits,Data]` come to mind) _are_ referred to by examples, etc. Is that something we _want_ to include in the rendered docs? cc @timhoffm, this is reminiscent of the discussions regarding `plt.Axes`/`plt.Figure`, but is different in that these are not imports and are instance attributes)

Reply:
`ax.title` is used in several tests as well as the `lifecycle` tutorial and `tight_layout_guide`

`axes.legend_` is used outside of the `axes` module only in `backend/qt_editor/figureoptions.py` as far as I can tell through quick greps (and never in documentation)